<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWin</class>
 <widget class="QMainWindow" name="MainWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sirius Drive</string>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>39</y>
      <width>961</width>
      <height>591</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>.AppleSystemUIFont</family>
     </font>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Downloads</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>332</x>
        <y>80</y>
        <width>171</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Download Files By Link</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>31</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>+</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_6">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>80</y>
        <width>81</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Manage</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>20</y>
        <width>161</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>.AppleSystemUIFont</family>
        <pointsize>15</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Downloads:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>24</x>
        <y>516</y>
        <width>101</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Refresh</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>12</x>
        <y>40</y>
        <width>502</width>
        <height>32</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>my_famaly_photos</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>movies</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>jazz_songs</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>191</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>.AppleSystemUIFont</family>
        <pointsize>15</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Channels:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_8">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>80</y>
        <width>31</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
     <widget class="QLabel" name="m_path">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>128</y>
        <width>505</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>.AppleSystemUIFont</family>
        <pointsize>15</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Path:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_36">
      <property name="geometry">
       <rect>
        <x>576</x>
        <y>516</y>
        <width>31</width>
        <height>32</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>.AppleSystemUIFont</family>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
     <widget class="QTableView" name="m_fsTreeTableView">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>156</y>
        <width>489</width>
        <height>345</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableView" name="m_downloadsTableView">
      <property name="geometry">
       <rect>
        <x>571</x>
        <y>47</y>
        <width>361</width>
        <height>453</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="m_downloadBtn">
      <property name="geometry">
       <rect>
        <x>523</x>
        <y>304</y>
        <width>33</width>
        <height>53</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>.AppleSystemUIFont</family>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>&gt;&gt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="DriveOwner">
     <attribute name="title">
      <string>Drives</string>
     </attribute>
     <widget class="QComboBox" name="comboBox_2">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>44</y>
        <width>431</width>
        <height>32</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Photos</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Movies</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Jazz_songs</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>24</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>.AppleSystemUIFont</family>
        <pointsize>15</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Drives:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_10">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>520</y>
        <width>151</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Apply Changes</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_11">
      <property name="geometry">
       <rect>
        <x>730</x>
        <y>80</y>
        <width>181</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Top Up Drive Account</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_12">
      <property name="geometry">
       <rect>
        <x>770</x>
        <y>520</y>
        <width>151</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Cancel Modification</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_15">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>80</y>
        <width>31</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>+</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_16">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>80</y>
        <width>31</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_17">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>80</y>
        <width>81</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Manage</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>128</y>
        <width>261</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>.AppleSystemUIFont</family>
        <pointsize>15</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Drive:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>28</x>
        <y>20</y>
        <width>261</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>.AppleSystemUIFont</family>
        <pointsize>15</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Difference:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_19">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>520</y>
        <width>171</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Sync With Replicators</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_18">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>10</y>
        <width>111</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Open In Files</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="m_diffView">
      <property name="geometry">
       <rect>
        <x>25</x>
        <y>45</y>
        <width>421</width>
        <height>457</height>
       </rect>
      </property>
      <property name="columnCount">
       <number>2</number>
      </property>
      <row>
       <property name="text">
        <string/>
       </property>
      </row>
      <row>
       <property name="text">
        <string/>
       </property>
      </row>
      <row>
       <property name="text">
        <string/>
       </property>
      </row>
      <row>
       <property name="text">
        <string/>
       </property>
      </row>
      <row>
       <property name="text">
        <string/>
       </property>
      </row>
      <column>
       <property name="text">
        <string/>
       </property>
      </column>
      <column>
       <property name="text">
        <string/>
       </property>
      </column>
      <item row="0" column="0">
       <property name="text">
        <string>file_1</string>
       </property>
      </item>
      <item row="0" column="1">
       <property name="text">
        <string>Add</string>
       </property>
      </item>
      <item row="1" column="0">
       <property name="text">
        <string>file_2</string>
       </property>
      </item>
      <item row="1" column="1">
       <property name="text">
        <string>Mod</string>
       </property>
      </item>
      <item row="2" column="0">
       <property name="text">
        <string>file_3</string>
       </property>
      </item>
      <item row="2" column="1">
       <property name="text">
        <string>Del</string>
       </property>
      </item>
      <item row="3" column="0">
       <property name="text">
        <string>file_4</string>
       </property>
      </item>
      <item row="3" column="1">
       <property name="text">
        <string>Add</string>
       </property>
      </item>
      <item row="4" column="0">
       <property name="text">
        <string>file_5</string>
       </property>
      </item>
      <item row="4" column="1">
       <property name="text">
        <string>Add</string>
       </property>
      </item>
     </widget>
     <widget class="QTreeView" name="m_driveTree">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>150</y>
        <width>441</width>
        <height>351</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>My Replicator</string>
     </attribute>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>30</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>.AppleSystemUIFont</family>
        <pointsize>15</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Drives:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>460</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>.AppleSystemUIFont</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Contributed 1024 GB</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>710</x>
        <y>460</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>.AppleSystemUIFont</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Used: 1024 GB</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QListView" name="listView_5">
      <property name="geometry">
       <rect>
        <x>35</x>
        <y>60</y>
        <width>889</width>
        <height>391</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_14">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>460</y>
        <width>31</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_20">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>240</y>
        <width>251</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Onboard Replicator</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="m_settingsBtn">
    <property name="geometry">
     <rect>
      <x>949</x>
      <y>10</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>.AppleSystemUIFont</family>
      <pointsize>32</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 10px;
color : #aa6600</string>
    </property>
    <property name="text">
     <string>⚙︎</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>10</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>10024 XPX</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>10</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Balance:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
